test/Filter/ORMFilterTest.php

```diff
@@ -35,7 +38,7 @@ public function setUp()
         parent::setUp();
 
         $serviceManager = $this->getApplication()->getServiceManager();
-        $objectManager = $serviceManager->get('doctrine.entitymanager.orm_default');
+        $this->objectManager = $objectManager = $serviceManager->get('doctrine.entitymanager.orm_default');
```

Comment:
I see you added an property for the objectManager, but the file still contains local variables in functions. For example the `private function countResult($filters, $entity = 'Db\Entity\Artist')`. Wouldn't it be nice to use the property aswell in there so the setup is the only one setting this.